ui/src/components/LandingPage.css

```diff
@@ -0,0 +1,11 @@
+.title {
+    text-align: center;
```

Comment:
```suggestion
    text-align: center;
    text-align: left;
```

Reply:
I didn't like it aligned to the left. But it is aligned left in the flat so I will do that.